### DISCUSSION TITLE
How to make Accordion / Accordion Content open by default ?

### DISCUSSION THREAD

**Discussion by 7kylor** [shadcn-ui/ui#1261] [Q&A]
I'm wondering if there is a way to make the  Accordion / Accordion Content open by default ?

**Comment by shadcn** [comment]
Use `type="single" and defaultValue="VALUE"`.

See example https://codesandbox.io/p/sandbox/jolly-smoke-943xdj?file=/components/accordion-demo.tsx:12,5 where `item-2` is expanded by default.

**Reply by 7kylor** [reply]
Thank You

**Comment by emekaokoli** [comment]
how about multiple items at once? type="multiple"?
